content/3_Silver/Prefix_Sums_2.mdx

```diff
@@ -439,6 +439,102 @@ int main() {
 
 </CPPSection>
 
+<JavaSection>
+
+```java
+import java.io.*;
+import java.util.StringTokenizer;
+
+public class ForestQueries {
+	static Kattio io = new Kattio();
+
+	public static void main(String[] args) throws IOException {
+		
+
+		int n = io.nextInt();
+		int q = io.nextInt();
+		int N = n + 1;
```

Comment:
```suggestion
		int N = io.nextInt()+1;
		int q = io.nextInt();
```

Reply:
add a space before and after the "+"
also, we can just do <=n instead of <N. i think that may make more sense.

Reply:
i think its fine, changing to to <= n is the same as < N anyways.

Reply:
can we have <= n?